src/components/organisms/Header.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBook, faBars } from '@fortawesome/free-solid-svg-icons';
+import UserAuthBtn from '../molecules/UserAuthBtn';
+
+function Header() {
+  const [collapse, setCollapse] = useState();
+  const [toggleClass, setToggleClass] = useState('');
+  const [toggleAria, setToggleAria] = useState('');
+  const [navClass, setNavClass] = useState('');
+
+  const updateToggleClass = (isActive) => {
+    if (isActive) {
+      setToggleClass('navbar-toggler collapsed');
+      setToggleAria('false');
+      setNavClass('navbar-collapse collapse');
+      return;
+    }
+    setToggleClass('navbar-toggler');
+    setToggleAria('true');
+    setNavClass('navbar-collapse collapse show');
+  };
+
+  useEffect(() => {
+    const initialToggleValue = true;
```

Comment:
Ummm, shouldn't this be the initial value for collapse `const [collapse, setCollapse] = useState(true);`;